packages/utilities/psammead-locales/moment/mr.js

```diff
@@ -3,4 +3,11 @@ require('moment/locale/mr');
 
 moment.updateLocale('mr', {
   postformat: null,
+  relativeTime: {
+    past: '%sपूर्वी',
```

Comment:
I believe there should be a space.
```suggestion
    past: '%s पूर्वी',
```